DataManagement/BranchTutorial.Rmd

```diff
@@ -85,87 +85,100 @@ knitr::include_graphics("Pics/6. add_url.png")
 
 ### Make new branch (RStudio)
+
 In RStudio, click on the New Branch button in the Git window. The box next to it will tell you in which branch you are. Master is the default and original branch.
 
 ```{r new-branch}
 knitr::include_graphics("Pics/7. NewBranch.png")
 ```
 <br/><br/>
-Give the branch a short and informative name.
+<br/><br/>
+Give the branch a short and informative name (no spaces) and click create.
 
 ```{r name-branch, out.width = '50%'}
 knitr::include_graphics("Pics/8. NameBranch.png")
 ```
 <br/><br/>
+<br/><br/>
 You can now see that you are in the new branch.
+
 ```{r new-branch-2}
 knitr::include_graphics("Pics/9. NewBranch2.png")
 ```
 <br/><br/>
+<br/><br/>
+
+### Write R code, commit and push to origin (Rstudio)
+
+Now is the time to open a new or an existing file and start to work on the project. Work in the usual way, add small bits of code, commit regularly, add a useful commit message and push once in a while (see here how to code with GitHub).
+Once you are done with your project, test it and push it to your orgin repository on GitHub. This works in the same way as when you work in your own repository.
 
-### Write R code (Rstudio)
-Now is the time to open a new or an existing file and start to write code. Work in the usual way, do small steps, commit regularly and push (see how to work in with git tutorial). 
 ```{r make-file}
 knitr::include_graphics("Pics/10. make file.png")
 ```
 <br/><br/>
-
-### Commit and push
-!!! This will be a separate tutorial!!! so very briefly described!!!
-write code, work in small logical steps (e.g. write a function, fix a function, add a plot, etc.). Do not make too many steps at a time. One step - commit - next step - commit -.... At the end of the day push.
 <br/><br/>
 
+
 ### Make pull request (github)
-Once you have pushed your branch back to the master branch you can make a pull request. Go back to Github, to the forked repository. A green box appears at the top of the page "pull requst". Click on this button.
+
+Once you have pushed your branch back to the master branch you can make a pull request. Go back to Github, to your forked repository. A green box appears at the top of the page "pull requst". Click on this button.
+<br/><br/>
 <br/><br/>
 ```{r pull-request-1}
 knitr::include_graphics("Pics/12 Pull request.png")
 ```
 <br/><br/>
-Add some comments about your pull request if necessary and create the pull request.
+<br/><br/>
+Add a comment about your pull request if necessary and create the pull request.
+<br/><br/>
 <br/><br/>
 ```{r pull-request-2}
 knitr::include_graphics("Pics/13. Pullrequest 2.png")
 ```
 <br/><br/>
-Once you have created your pull request, github will check if there are any merge conflict.
+<br/><br/>
+Once you have created your pull request, github will check if there are any merge conflict. If no merge conflicts occur, the box will become green. If there are merge conflicts, click on the box resolve merge conflicts and fix the problems.
```

Comment:
Somewhere we will have to explain what merge conflicts are and how to fix them

Reply:
I know...